lib/blockchain.js

```diff
@@ -0,0 +1,218 @@
+const dns = require('dns');
+const https = require('https');
+const fetch = require('node-fetch');
+const Web3 = require('web3');
+
+const {
+  ethereumHost,
+  infuraProjectId,
```

Comment:
Why is this getting the project id from some const, and elsewhere it gets it from env, and elsewhere it gets it from config?

Reply:
Oh I see the constants are not constants, they are pulled from the env.